generated/java/proto-google-longrunning-v1/staging.gradle

```diff
@@ -0,0 +1,42 @@
+buildscript {
```

Comment:
Since you only have the staging.gradle files so that you can declare project dependencies instead of maven dependencies, you should actually be able use a single build.gradle, since at release time, gradle will convert the project dependencies into maven dependencies.